src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/Intellisense.cs

```diff
@@ -204,10 +204,10 @@ protected static void TypeMatchPriority(DType type, IList<IntellisenseSuggestion
                 if (!suggestion.Type.IsUnknown && 
 
                     // Most type acceptance is straightforward
-                    (type.Accepts(suggestion.Type) ||
+                    (type.Accepts(suggestion.Type, exact: true, useLegacyDateTimeAccepts: false, usePowerFxV1CompatibilityRules: true) ||
```

Comment:
>true [](http://example.com/codeflow?start=129&length=4)

Is this always going to be true? Aren't we using any config to drive this value?

Reply:
Since this is intellisense (only for new or modified apps) we should be using the new values. But I'll pass the existing parameter along to minimize the number of changes for now.